frankenphp.go

```diff
@@ -311,7 +317,8 @@ func Init(options ...Option) error {
 
 	shutdownWG.Add(1)
 	done = make(chan struct{})
-	requestChan = make(chan *http.Request)
+	// keep latency down by having enough to concurrently handle requests, plus a small buffer
+	requestChan = make(chan *http.Request, opt.numThreads*2)
```

Comment:
If we have X threads, by making the channel at least X buffered, we can send X requests in parallel without blocking. So we set it to `numThreads*2` to allow each thread to have one ready to go without blocking, when under load.